docs/architecture.md

```diff
@@ -66,15 +66,15 @@ Core Workflow
         what │  |       |     ┌──▼──────┐
         data │  |       |     │Consensus│
           to │  |       |     └─*┌──────┘
-        sign │  |       |        │
-             │  |       |     ┌──▼───┐
-             │  |       |     │DutyDB│
-             │  |       |     └──▲───┘
-                |       |        │           │
-      *Sign* │  |       |     ┌──┴─┐         │
-        duty │  |       └----─┤VAPI◄─────────│── VC
-        data │  |             └──┬─┘         │ Query, sign, submit
-                |                │           │
+        sign │  |       |        ├────────────┐
+             │  |       |     ┌──▼───┐     ┌──▼───┐    │
+             │  |       |     │DutyDB│     │Signer├────│─► RS
```

Comment:
will there be any other responsibilities of `Signer` component apart from interaction with RS?

Reply:
I do not think so.